microk8s-resources/actions/disable.host-ip.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo "Disabling host-ip"
+
+if [ -f "${SNAP_DATA}/var/lock/host-ip-enabled" ]
+then
+  IP_ADDRESS=$(<"${SNAP_DATA}/var/lock/host-ip-enabled")
+  sudo ifconfig lo:17 $IP_ADDRESS down
```

Comment:
There is a utility function (in utils) called `run_with_sudo` that makes better handling of libs. Could you also make sure we bring ifconfig in the snap and call it from there (use full path under /snap/....)?

Reply:
done